file.go

```diff
@@ -204,6 +213,21 @@ func (db *DbFile) init() error {
 		def = strings.TrimSpace(def)
 
 		parts := strings.Split(def, ",")
+		// strip away statements like 'UNIQUE ...' or 'PRIMARY KEY ...' from a table definition
+		for i := range parts {
+			if i >= len(parts) {
+				break // we removed at least one elem, so avoid out of bounds read
+			}
+
+			parts[i] = strings.TrimSpace(parts[i])
+			for j := range idxtokens {
+				if strings.HasPrefix(parts[i], idxtokens[j]) {
+					// drop all other elements
+					parts = parts[:i]
```

Comment:
Is it safe to assume `UNIQUE` and/or `PRIMARY KEY` are always the last things printed in a table definition?

Reply:
No idea on the guarantees. The grammar would be the authoritative reference, and is pretty clear…

Reply:
Yea. I added the other table constraints from https://sqlite.org/lang_createtable.html